rds/variables.tf

```diff
@@ -0,0 +1,102 @@
+variable "allocated_storage_gb" {
+  type    = number
+  default = 20
+}
+
+variable "backup_retention_period_days" {
+  type    = number
+  default = 7
+}
```

Comment:
ide beirhatnak h azert 7 a default mert asszem az meg ingyer van

Reply:
I did not know that. 
I know 7 days is free for insights, but I have to double check on snapshots.

Reply:
I can't find references to this in the docs, could you please share ?

Reply:
yeah, becuase my original state is not true and I mixed it up with the another 7 days (inisghts probably) and probably this:
```
Q: Do I need to enable backups for my DB Instance or is it done automatically?

By default, Amazon RDS enables automated backups of your DB Instance with a 7 day retention period. Free backup storage is limited to the size of your provisioned database and only applies to active DB Instances. For example, if you have 100 GB of provisioned database storage over the month, we will provide 100 GB-months of backup storage at no additional charge.
```